### ISSUE TITLE
Enhance i18n error messages

### ISSUE THREAD

**Issue by alex-bezek** [cerner/terra-toolkit#95]
# Issue Description
Translation errors are misleading. Update to give a more meaningful error message

## Issue Type

- [ ] New Feature
- [X] Enhancement
- [ ] Bug
- [ ] Other

## Expected Behavior
When there is an error getting translations generated, the exception message should be given to the user.

## Current Behavior
We swallow the exception and just tell them the file is not found which is not always correct.
An engineer struggled with this for a while because the file had a trailing comma. If the exception had been given instead,
debugging would have been easier

## Steps to Reproduce
1. Add an extra trailing comma in your translation file
2. Run your commands to generate translations (for us its booting webpack)
3. See the output `Translation file en.json not found for /Users/as027811/code/hlife/education/translations`
4. See that this file exists 
5. Be confused and cry on the inside
```
➜  education git:(master) ✗ cat /Users/as027811/code/hlife/education/translations/en.json 
{
  "Education.I18nTest.message": "Test translation loaded from translations/en.json",
}

```

## Proposed Fix
Change the console warning to something like
`console.warn(`There was an error reading your translations file ${translationFile}. \n Exception Message: ${e.message} \n`)`

and then you getting errors like
```
There was an error reading your translations file /Users/as027811/code/hlife/education/translations/en.json. 
 Exception Message: Unexpected token } in JSON at position 87 

There was an error reading your translations file /Users/as027811/code/hlife/education/translations/en-US.json. 
 Exception Message: ENOENT: no such file or directory, open '/Users/as027811/code/hlife/education/translations/en-US.json' 
 ```
 NOTE: the second file is actually not found, so the old error message still works basically
 ```
 ➜  education git:(master) ✗ ll /Users/as027811/code/hlife/education/translations
total 16
-rw-r--r--  1 as027811  WHQ_NT_DOMAIN\Domain Users  225 May  2 09:39 en-US-2.json
-rw-r--r--  1 as027811  WHQ_NT_DOMAIN\Domain Users   90 May  2 09:47 en.json
```

**Comment by bjankord** [comment]
+1 to making the error messages more helpful

**Comment by emilyrohrbough** [comment]
Resolved in #97
